### DISCUSSION TITLE
LED not working

### DISCUSSION THREAD

**Discussion by bunglesss** [klaasnicolaas/home-assistant-glow#433] [General]
Hi,

I flashed the software to the device as shown in the instructions (had to use Chrome on Linux, Brave wouldn't work).

It booted okay, and I bound it to my WIFI and added to Home Assistant.

However, I tried wiring the LED on, and it doesn't work. Have I bought the right things?

https://www.aliexpress.com/item/1005003302215339.html?spm=a2g0o.order_list.order_list_main.30.3f421802s75pWA
https://www.aliexpress.com/item/4000225253691.html?spm=a2g0o.order_list.order_list_main.40.3f421802s75pWA
https://www.aliexpress.com/item/1005005970816555.html?spm=a2g0o.order_list.order_list_main.35.3f421802s75pWA

Not tried the Diode yet, as ran out of connector leads :)

**Comment by klaasnicolaas** [comment]
The question is whether you purchased a cathode or anode variant of the LED 😉 

./Klaas

**Reply by bunglesss** [reply]
Ahhh, crap, which do I need? The packet has a big "A" written on it?

**Reply by klaasnicolaas** [reply]
You need one with a common **cathode**, just assume that you ordered a common **anode**.

![image](https://github.com/klaasnicolaas/home-assistant-glow/assets/20448157/2189bfba-d4de-47b9-bbca-7df6c3036499)

**Reply by bunglesss** [reply]
Baaaalls, what a muppet, no wonder it didn't work haha! Thanks mate, have ordered the same ones, but Cathode. It said "Foggy" but don't suppose that will matter.

**Comment by bunglesss** [comment]
Is the bulb looking bit on this the bit you should put over the flashing LED on the electric meter?

https://www.aliexpress.com/item/1005003302215339.html?spm=a2g0o.order_list.order_list_main.30.3f421802s75pWA

I just stuck that over the LED with some gaffer tape to test it. Don't think its working. Do I need to adjust that pot on it?

**Reply by klaasnicolaas** [reply]
Yes, that looks like the correct PCB board with a photodiode. Don't you have a photo of the situation?

Every time it measures a pulse, a small LED on the PCB will light up. You can also test this well, for example, by simulating a flash with a flashlight.

**Comment by bunglesss** [comment]
![IMG20240501181405](https://github.com/klaasnicolaas/home-assistant-glow/assets/45080520/5e7cc3a7-0cb4-4c42-85c7-4259d231bc0b)
![IMG20240501181333](https://github.com/klaasnicolaas/home-assistant-glow/assets/45080520/397492c3-f044-4a0a-a367-3afdab7ede97)

**Comment by bunglesss** [comment]
As you can see, they're the other side of the wall from each other, so that's a bit of a pain.

**Comment by bunglesss** [comment]
Is the LED on the smaller board with the photodiode supposed to flash when it gets a red pulse? It seems to be lit up green solid.

Then there's a red LED on the main board in the garage, and that also seems to always be on.
